lib/index.js

```diff
@@ -41,8 +41,13 @@ function toFloat(v) {
     return parseFloat(v, 10);
 }
 
+function round(num) {
+    const factor = Math.pow(10, 10);
+    return Math.round(num * factor) / factor;
+}
+
 function coordEqual(c1, c2) {
-    return x(c1) === x(c2) && y(c1) === y(c2);
+    return round(x(c1)) === round(x(c2)) && round(y(c1)) === round(y(c2));
```

Comment:
Instead of rounding, why not compare the absolute value of the difference?

```
Math.abs(x(c1) - x(c2)) < 1e-10
```

That's AFAIK the proper way to do comparison of floats. Using a multiplication means you risk an overflow when dealing with large numbers. Also quite a bit faster.

Reply:
Good idea! I will do some changes.

Reply:
Done!